recipe/meta.yaml

```diff
@@ -13,25 +13,26 @@ source:
   patches:
     # ctypes.util.find_library() isn't CONDA_PREFIX aware :'(
     # https://github.com/ContinuumIO/anaconda-issues/issues/1716
-    - 0001-Make-load_lib-CONDA_PREFIX-aware.patch  # [unix]
+    - patches/0001-Make-load_lib-CONDA_PREFIX-aware.patch  # [unix]
 
 build:
   number: 0
-  script: '{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv '
+  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv --no-build-isolation
 
 requirements:
   build:
     - {{ compiler('c') }}
     - patch       # [not win]
-    - m2-patch    # [win]
   host:
     - python
     - pip
     - setuptools
     - wheel
+    - gmp   # [not win]
   run:
     - python
-    - gmp   # [not win]
+    - gmp  # [not win]
+
```

Comment:
Don't need this since `gmp` has a `run_export`. It should be applied for you.
https://conda-metadata-app.streamlit.app/?q=pkgs%2Fmain%2Flinux-64%2Fgmp-6.2.1-h295c915_3.conda&with_broken=true